frontend/src/metabase/embedding/embedding-iframe-sdk-setup/components/GetCodeStep.tsx

```diff
@@ -45,62 +46,71 @@ export const GetCodeStep = () => {
 
   const handleServerSnippetCopied = () => trackSnippetCopied("server");
 
+  // We center content when embedding is not enabled to show empty state text
+  const justify = !resource?.enable_embedding ? "center" : undefined;
```

Comment:
Is there a way to just use pure CSS? e.g. instead of centering the child, if we scale the child to 100% height and have the child center itself, would it work?

I just don't like using JS for styling if there's a way to use pure CSS as it's much easier to grasp too. Mixing JS and CSS is ok when you need dynamic values e.g. runtime content dimension. But I feel like there could be a simple pure CSS solution here.

Reply:
Adjusted